boards/arm/nucleo_l011k4/nucleo_l011k4.dts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 Steven Daglish <s.c.daglish@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ dts - v1 /;
+#include <st/l0/stm32l011X4.dtsi>
+#include <st/l0/stm32l011k(3-4)tx-pinctrl.dtsi>
+#include "arduino_r3_connector.dtsi"
+
+/ {
+	model = "STMicroelectronics STM32L011K4-NUCLEO board";
+	compatible = "st,stm32l011k4-nucleo";
+
+	chosen {
+		zephyr, console = &usart2;
+		zephyr, shell - uart = &usart2;
```

Comment:
```suggestion
		zephyr, shell-uart = &usart2;
```
Please fix in the whole file.

Reply:
Fixed all instances in file.

Reply:
I found a few more formatting errors that occurred after using uncrustify, which have now also been fixed.